packages/extension/src/view/devtools/components/tabContent/cookies/cookieFilter/chips/chip.tsx

```diff
@@ -25,7 +25,10 @@ import type {
   SelectedFilters,
   Filter,
 } from '../../../../../stateProviders/filterManagementStore/types';
-import { FILTER_MAPPING } from '../../../../../stateProviders/filterManagementStore/constants';
+import {
+  CUSTOM_FILTER_MAPPING,
+  FILTER_MAPPING,
+} from '../../../../../stateProviders/filterManagementStore/constants';
```

Comment:
Could you please add types and constants to the index file of filter management store so these can be imported with one statement